pkg/alerts/policies.go

```diff
@@ -193,6 +197,32 @@ func (a *Alerts) QueryPolicy(accountID, id int) (*QueryPolicy, error) {
 	return &resp.Actor.Account.Alerts.Policy, nil
 }
 
+// QueryPolicySearch searches NerdGraph for policies.
+func (a *Alerts) QueryPolicySearch(accountID int, params AlertsPoliciesSearchCriteriaInput) ([]QueryPolicy, error) {
```

Comment:
I think we've been returning an array of pointers - i.e. `[]*QueryPolicy` - for consistency

Reply:
Done.